std/experimental/allocator/package.d

```diff
@@ -1601,6 +1601,125 @@ unittest //bugzilla 15721
 }
 
 /**
+Allocates a multidimensional array of elements of type T.
+
+Params:
+N = number of dimensions
```

Comment:
I thought that this is common for statically defined arrays [1][2]

[1] https://dlang.org/library/std/experimental/ndslice/selection/iota_slice.html
[2] https://github.com/dlang/phobos/blob/master/std/experimental/ndslice/slice.d#L834